src/imitation/scripts/config/parallel.py

```diff
@@ -8,40 +8,37 @@
 search spaces to the config like `"seed": tune.grid_search([0, 1, 2, 3])`.
 """
 
-import os.path as osp
-
 import ray.tune as tune
 import sacred
 
-import imitation.util as util
-
 parallel_ex = sacred.Experiment("parallel")
 
 
 @parallel_ex.config
 def config():
   inner_experiment_name = "expert_demos"  # The experiment to parallelize
+  inner_run_name = "DEFAULT"  # CLI --name option. Used for analysis grouping.
```

Comment:
Might be better if it defaulted to a random UUID, rather than to a fixed value? (We'll end up with a lot of things with "DEFAULT" because we forgot to set a run name.)

Reply:
Good idea, added UUID